dev-ruby/http/http-5.0.4.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+USE_RUBY="ruby27 ruby30 ruby31"
+
+RUBY_FAKEGEM_RECIPE_TEST="rspec3"
+RUBY_FAKEGEM_EXTRADOC="CHANGES.md README.md"
+RUBY_FAKEGEM_GEMSPEC="${PN}.gemspec"
+# disabled for now.  pulls in another large dep tree for ruby31
+# RUBY_FAKEGEM_RECIPE_DOC="yard"
+inherit ruby-fakegem
+
+DESCRIPTION="An easy-to-use client library for making requests from Ruby"
+HOMEPAGE="https://github.com/tarcieri/http"
+
+LICENSE="MIT"
+SLOT="5"
+KEYWORDS="~amd64 ~sparc"
+IUSE=""
+
+ruby_add_rdepend ">=dev-ruby/addressable-2.8:2
+	>=dev-ruby/http-cookie-1.0:1
+	>=dev-ruby/http-form_data-2.2:2
+	>=dev-ruby/llhttp-ffi-0.4.0:0/0.4"
+
+ruby_add_bdepend "
+	test? (
+		>=dev-ruby/certificate_authority-1.0:1
+		dev-ruby/rspec-its
+		dev-ruby/simplecov
```

Comment:
In general we try to avoid developer dependencies such as coverage to keep the dependency trees more manageable. We are only interested in the tests. So the preference here is to patch out things like coverage.